src/plots/linePlot.ts

```diff
@@ -21,6 +22,28 @@ export module Plots {
       this.attr("stroke-width", "2px");
     }
 
+    /**
+     * Gets the interpolation function associated with the plot.
+     *
+     * @return string | (points: Array<[number, number]>) => string)
+     */
+    public interpolate(): string | ((points: Array<[number, number]>) => string);
+    /**
+     * Sets the interpolation function associated with the plot.
+     *
+     * @param interpolate Interpolation function
+     * @return Plots.Line
+     */
+    public interpolate(interpolate: string | ((points: Array<[number, number]>) => string)): Plots.Line<X>;
```

Comment:
Since "interpolate" is a verb, in a vacuum I would prefer to name this method "`interpolator()`". However, d3 uses "`interpolate`". I think it's okay to depart from D3 here.

Also, is it worth waiting for the bug mentioned [here](https://github.com/borisyankov/DefinitelyTyped/pull/5095#issuecomment-125683305) to get fixed in D3, so we don't have to return a union type?